frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/header/cover_title.dart

```diff
@@ -0,0 +1,225 @@
+import 'package:appflowy/generated/locale_keys.g.dart';
+import 'package:appflowy/plugins/document/application/document_appearance_cubit.dart';
+import 'package:appflowy/workspace/application/appearance_defaults.dart';
+import 'package:appflowy/workspace/application/view/view_bloc.dart';
+import 'package:appflowy_backend/protobuf/flowy-folder/view.pb.dart';
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+FocusNode? coverTitleFocusNode;
```

Comment:
I'm not a huge fan of this dangling global variable. Can we use an inherited widget or a Provider?

Reply:
Let me see if there is a way to share this value. Using a provider or inherited widget doesn't work here unless we register this value at the top of the widget tree.